src/master/develop/management.md

```diff
@@ -72,8 +72,98 @@ immuClient.changePassword(username, password, "newTestTest123!");
 :::
 
 ::: tab Python
-This feature is not yet supported or not documented.
-Do you want to make a feature request or help out? Open an issue on [Python sdk github project](https://github.com/codenotary/immudb-py/issues/new)
+```python
+from grpc import RpcError
+from immudb import ImmudbClient
+from enum import IntEnum
+
+URL = "localhost:3322"  # immudb running on your machine
+LOGIN = "immudb"        # Default username
+PASSWORD = "immudb"     # Default password
+DB = b"defaultdb"       # Default database name (must be in bytes)
+
+class Permission(IntEnum):
+    NONE = 0
```

Comment:
Hmm, those enums should be part of the SDK - please add a gh issue for that.

Reply:
https://github.com/codenotary/immudb-py/issues/38